app/views/shared/_header.html.erb

```diff
@@ -1,33 +1,43 @@
 <nav class="navbar navbar-default">
   <div class="container-fluid">
-<% unless logged_in? %>
-    <p class="navbar-text">Not signed in</p>
-<% else %>
+  <% unless logged_in? %>
+      <p class="navbar-text">Not signed in</p>
+  <% else %>
+
+    <% # Links menu will be empty when an external approver comes in.%>
+    <% # In that case we don't display any of the navigation bits. %>
+    <% nav_menu = [] %>
```

Comment:
This is the kind of thing that preferably would be pulled out into a presenter, but this logic should get pulled out into a presenter, but since this is shared I'm not sure exactly where it would go. Might be a good candidate for a top-level helper. I'll review a bit and see if I can come up with something here.

Reply:
I think a helper is actually the way to go here, since (unlike with a presenter) this isn't really wrapping presentation of a specific AR model.

Reply:
I suppose the controller could provide a `@nav_menu` object that was a fully-fledged thing, but that seems like overkill for the moment..